bin/riemann/deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -x
+
+. ./bin/cloud-common.sh
+
+machine_name=$(swarm_master_name)
+
+eval $(docker-machine env --swarm "$machine_name")
+
+docker-compose -f ./config/riemann/cluster.yml pull
+
+docker-compose -f ./config/riemann/cluster.yml up -d
+
+docker-compose -f ./config/riemann/cluster.yml scale riemannhealth=5
```

Comment:
With this we are scaling only the riemannhealth service, that is, the clients that run on each node. We need as much of this clients as nodes we have, they will send metrics to the central server. We only need one instance of the services, riemannserver and riemanndash.